src/transformer/descriptor/mock/mockProperties.ts

```diff
@@ -44,6 +44,10 @@ export function GetMockPropertiesFromDeclarations(
         return false;
       }
 
+      if (member.questionToken) {
```

Comment:
This is cool. I think we can safely remove the code below from file propertySignature.ts

```
if (node.type) {
    if (node.questionToken) { // remove this if
      return GetUndefinedDescriptor();
    }

    return GetDescriptor(node.type, scope);
  }
```

What do you think?

Reply:
it makes sense, we exclude it early, it should never get to that point if it has the question token, I'll remove it later today

Reply:
done